### PR TITLE
Accessibility fixes

### PR DESCRIPTION
**Pull request by edmadrid**
https://issues.cul.columbia.edu/browse/ACHYDRA-706

### CODE REVIEW

File: app/assets/javascripts/skip-link-focus-fix.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+$( document ).ready(function() {
```
Comment (cgalarza) [inline app/assets/javascripts/skip-link-focus-fix.js]: Can you add a quick description of what this is for? Just so its documented with the code?

File: app/views/uploads/new.html.erb [hunk 2]
```diff
@@ -58,10 +58,10 @@
         <div id='creator-list'>
           <% (@deposit.creators || [{}]).each do |c| %>
```
Comment (cgalarza) [inline app/views/uploads/new.html.erb]: This block repeats so we probably shouldn't use static names for id's one thing we could do is change the line above to be:

```suggestion
          <% (@deposit.creators || [{}]).each_with_index do |c, i| %>
```

File: app/views/uploads/new.html.erb [hunk 3]
```diff
@@ -58,10 +58,10 @@
...
             <div>
-              <input type="text" name="deposit[creators][][first_name]" class="deposit_creators_first_name", value="<%= c['first_name'] %>">
-              <input type="text" name="deposit[creators][][last_name]" class="deposit_creators_last_name", value="<%= c['last_name'] %>">
-              <input type="text" name="deposit[creators][][uni]" class="deposit_creators_uni",  value="<%= c['uni'] %>" <%= 'readonly="readonly"' if c['uni'] == current_user.uid %>>
-              <span class="sort-icon"><i class="fas fa-arrows-alt" title="Sort creators" aria-hidden></i></span><span class="visually-hidden">Sort creators</span>
+              <input type="text" name="deposit[creators][][first_name]" id="deposit_creators_first_name", class="deposit_creators_first_name", value="<%= c['first_name'] %>">
```
Comment (cgalarza) [inline app/views/uploads/new.html.erb]: and change the ids to be something like
```suggestion
              <input type="text" name="deposit[creators][][first_name]" id="<% "deposit_creators_first_name_#{i}"%>", class="deposit_creators_first_name", value="<%= c['first_name'] %>">
```

File: app/assets/javascripts/skip-link-focus-fix.js (hunk 1 above)
Comment (edmadrid) [reply app/assets/javascripts/skip-link-focus-fix.js]: added description to js file

**Comment by edmadrid** [comment]
@cgalarza I forgot that you could add new creators. I swapped things around so that the labels have an ID, and then the creator list inputs each have "aria-labelledby=". This passes accessibility-wise. Does this work?

**Comment by cgalarza** [comment]
@edmadrid Your changes look good!

**Pull request merged**